pybedlite/overlap_detector.py

```diff
@@ -333,15 +338,21 @@ def overlaps_any(self, interval: Span) -> bool:
             # to start
             return tree.any_overlaps(interval.start + 1, interval.end)
 
-    def get_raw_overlaps(self, interval: Span) -> list[SpanType]:
```

Comment:
I don't see how removing this makes the PR _better_. If you dislike the name, then we can could change it. But I don't feel like it's worth holding up the PR over.

Reply:
I don't want to add another method to side-step the the superfluous processing in `get_overlaps()`.

Reply:
Extra reasoning here:

- https://github.com/fulcrumgenomics/pybedlite/pull/42#discussion_r2361120321